conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureSafeLongList.java

```diff
@@ -56,6 +56,14 @@ public boolean addAll(int index, Collection<? extends SafeLong> collection) {
         return delegate.addAllAtIndex(index, target);
     }
 
+    public void addAll(long... source) {
+        for (long value : source) {
+            // Doesn't use SafeLong creation because this causes unnecessary boxing
```

Comment:
This comment is a little too specific and doesn't explain how the underlying type holds longs and those must adhere to the abstraction.